include/net/xia_ether.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
```

Comment:
The reason the HID principal has an include file is that it is broken into two files: `ppal_hid/main.c` and `ppal_hid/nwp.c`. The Ethernet principal is simpler, combining `xia_ether.h` and `ppal_ether/main.c` will create a file with 362 + 833 = 1195 lines. That is a little over the number of lines in `ppal_hid/nwp.c`: 1031 lines.

In other words, I think this whole file can move inside of `ppal_ether/main.c`. This is not unusual, as you can check, the AD principal doesn't have an include file as well.

Reply:
Should i let the data structures remain inside the header file?

Reply:
No, because they are not being used outside of `net/xia/ppal_ether/main.c`.

What `include/net/xia_ether.h` is exporting now is fine. Thank you.